core/src/main/res/layout/mz_view_map.xml

```diff
@@ -30,16 +30,46 @@
       android:layout_width="@dimen/mz_find_me_button_width"
       android:layout_height="@dimen/mz_find_me_button_height"
       android:layout_alignParentRight="true"
-      android:layout_alignParentBottom="true"
+      android:layout_alignParentTop="true"
       android:layout_marginRight="@dimen/mz_find_me_button_margin_right"
-      android:layout_marginBottom="@dimen/mz_find_me_button_margin_bottom"
+      android:layout_marginTop="@dimen/mz_find_me_button_margin_top"
       android:background="@drawable/mz_bg_white_gray_border"
       android:visibility="gone"
       android:src="@drawable/mz_find_me"
       android:contentDescription="@string/mz_find_me"
       tools:ignore="RtlHardcoded"
       />
 
+  <ImageButton
+      android:id="@+id/mz_zoom_in"
```

Comment:
Yes, I didn't noticed it. I tried to add `android:scaleType="center"` there but it didn't helped. I have also tried `android:scaleType="fitCenter"` but it didn't helped either. I have also reviewed size of that zoom out button image and it's width and height size is uniform. So I don't know why this is happening right now. But minus button is ok.